streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordDeserializer.java

```diff
@@ -113,6 +115,7 @@ public static void handleDeserializationFailure(final DeserializationExceptionHa
             throw new StreamsException("Deserialization exception handler is set to fail upon" +
                 " a deserialization error. If you would rather have the streaming pipeline" +
                 " continue after a deserialization error, please set the " +
+                DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG + " or the deprecated exception handler " +
```

Comment:
We should just point to the new one and not refer to the deprecated one at all. (For this case, we also don't need the `@SuppressWarnings` annotation